examples/web-instrumentation/browser/src/LoadTest.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Observable} from 'rxjs';
+import {ajax} from 'rxjs/ajax';
+import {LogCollectorBuilder} from './lib/LogCollectorBuilder';
+import {TestPayload} from './TestPayload';
+import {TestResults} from './TestResults';
+import {CollectorExporter} from '@opentelemetry/exporter-collector';
+import {SimpleSpanProcessor} from '@opentelemetry/tracing';
+import {XMLHttpRequestPlugin} from '@opentelemetry/plugin-xml-http-request';
+import {ZoneScopeManager} from '@opentelemetry/scope-zone';
+import {WebTracerProvider} from '@opentelemetry/web';
+
+/**
+ * Class to run a load test and track reuqests.
+ */
+export class LoadTest {
+  /**
+   * Create a LoadTest
+   * @param {string} name - The name of the test
+   * @param {number} targetIter - The target number of iterations
+   * @param {number} delay - Time between requests
+   * @param {string} collectorURL - The URL of the OpenCensus agent
+   */
+  constructor(name, targetIter, delay, collectorURL) {
+    this.name = name;
+    this.targetIter = targetIter;
+    this.delay = delay;
+    this.buildId = 'v0.01';
+    this.numSent = 0;
+    this.numSuccess = 0;
+    this.numFail = 0;
+    const builder = new LogCollectorBuilder().setBuildId(this.buildId);
+    this.logCollector = builder.makeLogCollector();
+    this.logCollector.start(2000);
+    const webTracerWithZone = new WebTracerProvider({
+      // defaultAttributes: {'version': __VERSION__, 'test': name},
+      scopeManager: new ZoneScopeManager(),
+      plugins: [
+        new XMLHttpRequestPlugin({
+          ignoreUrls: ['/log', '/trace'],
+        }),
+      ],
+    });
+    const collectorOptions = {
+      url: collectorURL,
+    };
+    const exporter = new CollectorExporter(collectorOptions);
+    webTracerWithZone.addSpanProcessor(new SimpleSpanProcessor(exporter));
```

Comment:
Please add a note here: **It is recommended to use this BatchSpanProcessor (instead of SimpleSpanProcessor) for better performance and optimization, especially in production.**

Reply:
Changed code to use BatchSpanProcessor